docs/testing/testplan.md

```diff
@@ -0,0 +1,427 @@
+# Web Monetization Extension Test Plan
+
+## Introduction
+
+The purpose of this document is to guide end-user testing of the Web Monetization browser extension.
+We focus on functional test cases. The goal is to report bugs, issues or differences observed from the expected behavior.
+
+## Audience
+
+Any person interested in testing the beta release of the Web Monetization extension.
+
+## Objectives
+
+1. Validating that, once installed, users can configure Web Monetization settings on the extension.
+2. Validating that once the extension is successfully connected to a digital wallet, a user is able to configure payments or disconnect from the wallet.
+3. When the extension facilitates paying a monetized web page, check how payments get distributed between the receiving wallets on the page.
```

Comment:
Objective 3 relates to an edge test case. I'm not sure if we should call it out here. Just pointing it out, you can do what you think makes the most sense 🙂

Reply:
That's a fair point. I reworded objective 3: 
> Validating that the behaviour of the extension meets expectations, and that it is correct when the extension facilitates paying a monetized web page.